client/state/jetpack-connect/actions.js

```diff
@@ -44,11 +44,27 @@ export default {
 		};
 	},
 
-	checkUrl( url ) {
+	checkUrl( url, isUrlOnSites ) {
 		return ( dispatch ) => {
 			if ( _fetching[ url ] ) {
 				return;
 			}
+
+			if ( isUrlOnSites ) {
+				dispatch( {
+					type: JETPACK_CONNECT_CHECK_URL,
+					url: url,
+				} );
+				setTimeout( () => {
+					dispatch( {
+						type: JETPACK_CONNECT_CHECK_URL_RECEIVE,
+						url: url,
+						data: { exists: true, isWordPress: true, hasJetpack: true, isJetpackActive: true, isJetpackConnected: true, isWordPressDotCom: false, userOwnsSite: true },
```

Comment:
done! ( in jetpack-connect/index.jsx, where isUrlOnSites will only be set as true if the site is a jetpack one)